Lombiq.NodeJs.Extensions/scripts/run-pnpm-script.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs');
+const path = require('path');
+const { execSync, exec } = require('child_process');
+
+const { handleErrorObject } = require('./handle-error');
+
+function panic(message) {
+    handleErrorObject(message);
+    process.exit(1);
+}
+
+// Check if pnpm is installed.
+try { execSync('pnpm -v'); }
+catch (_) { panic('This project requires pnpm. You can install it with the "npm install --global pnpm" command.'); }
```

Comment:
Let's link to the prerequisites section in the documentation. Currently, `pnpm@6` is required due to some incompatibilities with `v7`. This will likely change soon, so let's not burden ourselves with duplicate work later on.